metrics/influxdb/influxdb.go

```diff
@@ -193,27 +193,27 @@ func (r *reporter) send() error {
 				},
 				Time: now,
 			})
-		case metrics.ResettingTimer:
-			t := metric.Snapshot()
-
-			if len(t.Values()) > 0 {
-				ps := t.Percentiles([]float64{50, 95, 99})
-				val := t.Values()
-				pts = append(pts, client.Point{
-					Measurement: fmt.Sprintf("%s%s.span", namespace, name),
-					Tags:        r.tags,
-					Fields: map[string]interface{}{
-						"count": len(val),
-						"max":   val[len(val)-1],
-						"mean":  t.Mean(),
-						"min":   val[0],
-						"p50":   ps[0],
-						"p95":   ps[1],
-						"p99":   ps[2],
-					},
-					Time: now,
-				})
-			}
+			//case metrics.ResettingTimer:
```

Comment:
Could you please explain the reason why you commented out these lines?

Reply:
`metrics.ResettingTimer` is a newly added type in the forked version of `metrics` package, however, this is the only place it is used. As original metrics package, `github.com/rcrowley/go-metrics` does not have `metrics.ResettingTimer`, I commented out these lines. It would be better to remove these lines instead of comment them out.